spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -13,18 +13,37 @@
 import pdb
 import re
 
+from qtpy.QtCore import Qt
```

Comment:
Please move this line to be below this one:

    from qtconsole.rich_jupyter_widget import RichJupyterWidget